repo/config/init.go

```diff
@@ -120,7 +120,10 @@ func Init() (*Config, error) {
 					"X-Textile-Opts",
 					"X-Requested-With",
 				},
-				"Access-Control-Allow-Origin": {},
+				"Access-Control-Allow-Origin": {
```

Comment:
another option here would be [], which then means all origins are ok. honestly not a bit deal given cors is really only a browser thing anyway... but this feels a bit more 'locked down' than all origins